### ISSUE TITLE
Add Rule S6304 from Sonar default profile

### ISSUE THREAD

**Issue by victor-diez-sonarsource** [SonarSource/SonarJS#3525]
Once the new RSPEC is done considering the cases reported in:

https://discuss.sonarsource.com/t/python-s6304-granting-access-to-all-resources/12137

APPSEC ticket:

https://sonarsource.atlassian.net/browse/APPSEC-219

**Comment by Wohops** [comment]
Migrated to Jira. New ticket number: JS-112
